tests/driver_lsm6dsl/README.md

```diff
@@ -3,8 +3,43 @@ This is a manual test application for the ST LSM6DSL sensor driver.
 
 # Usage
 
-After successful initialization, the sensor reads the accelerometer, gyroscope,
-and temperature values every 500ms and prints them to the STDOUT. If device
-initialization fails the tests exits with 1, otherwise it runs forever.
+After successful initialization the following tests are done :
+    * Acc power down
+    * Gyro power down
+    * wait 1 second
+    * Acc power up
+    * Gyro power up
+    * Enable latched interrupt
+    * Disable latched interrupt
+    * Read Acc
+    * Read Gyro
+    * Read Temp
+
+If all previous steps are OK, a shell is launched and the user can do several
+things with the `lsm6dsl` command. For the following commands/examples you can
+read the `usage` by entering the command needed without parameters.
+
+Here are some examples:
+
+ * Read acc value :
+```
+    lsm6dsl read acc
+```
+ * Configure wake up interrupt (adapt parameters values to your usage):
+```
+lsm6dsl configure  wkup_int 0 2 1 0
```

Comment:
The README is only showing an example usage of the `wkup_int` subcommand, it would be even more useful to show examples for the other sub-commands (`sleep_int`, `ff_int`, etc).

```suggestion
lsm6dsl configure wkup_int 0 2 1 0
```